sources/deny.toml

```diff
@@ -56,21 +56,27 @@ license-files = [
 multiple-versions = "deny"
 wildcards = "deny"
 
+deny = [
+    { name = "structopt" },
+    { name = "clap", wrappers = [ "cargo-readme" ] },
+]
+
 skip = [
 ]
 
 skip-tree = [
     # hyper-proxy is using an older hyper-rustls
     { name = "hyper-proxy", version = "=0.9.1" },
-    # structopt pulls in an older version of clap
-    { name = "structopt", version = "=0.3.26" },
     # tungstenite is using an older sha-1
     { name = "tungstenite", version = "=0.16" },
     # windows-sys is not a direct dependency. mio and schannel
     # are using different versions of windows-sys. we skip the
     # dependency tree because windows-sys has many sub-crates
     # that differ in major version.
     { name = "windows-sys", version = "=0.42.0" },
+    # generate-readme pulls in an older clap that causes some duplicate
+    # dependencies
+    { name = "clap", version = "=2.34.0" }
```

Comment:
Rather than skipping `clap`'s tree, I would just skip the tree for `generate-readme` itself.

Reply:
Done!